src/js/components/development/index.jsx

```diff
@@ -32,5 +34,26 @@ export default {
                 </div>
             </div>
         </div>
-    )
+    ),
+    
+    Bearer: () => {
+      const { loading, isAuthenticated, getTokenSilently } = useAuth0();
+      const [token, setToken] = useState("");
+
+      if (loading) {
+        return <Simple>Loading...</Simple>;
+      }
+
+      if (!isAuthenticated) {
+        return (
+          <Simple>
+            Not authenticated!
+          </Simple>
+        );
+      }
+
+      (async () => setToken(await getTokenSilently()))();
+
+      return <Simple><pre class="bearer-token">{token}</pre></Simple>;
```

Comment:
It must not be used `class`. Use `className` instead. :man_shrugging: 
```suggestion
      return <Simple><pre className="bearer-token">{token}</pre></Simple>;
```

Reply:
Hell why???

Reply:
because https://github.com/facebook/react/issues/13525#issuecomment-417818906
![image](https://user-images.githubusercontent.com/2437584/75285833-a8efc780-5817-11ea-9f7b-6a3a99af4259.png)